bors.toml

```
@@ -1,5 +1,5 @@
 status = [
-  "CLIMA",
+  "ClimateMachine.jl",
```

```suggestion
  "azure-ci",
```
https://dev.azure.com/climate-machine/ClimateMachine.jl/_build?definitionId=13&_a=summary

---

Done.